internal/pkg/warewulfd/api/overlay.go

```diff
@@ -288,3 +288,37 @@ func deleteOverlayFile() usecase.Interactor {
 	u.SetTags("Overlay")
 	return u
 }
+
+func updateOverlay() usecase.Interactor {
```

Comment:
This should be `updateOverlayFile()`, with similar edits throughout the function.